content/en/docs/tasks/configure-pod-container/static-pod.md

```diff
@@ -230,11 +229,11 @@ The running kubelet periodically scans the configured directory (`/etc/kubelet.d
 #
 mv /etc/kubelet.d/static-web.yaml /tmp
 sleep 20
-docker ps
+crictl ps
 # You see that no nginx container is running
 mv /tmp/static-web.yaml  /etc/kubelet.d/
 sleep 20
-docker ps
+crictl ps
```

Comment:
Does the switch to `crictl` change the output format? If so, we should revise that too.

Reply:
Resolved.